Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqProject.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-GcbqProject" {
+
+	It "should return the list of projects that the user has permissions to view" {
+		$batches = Get-GcbqProject
```

Comment:
Do we know how to create a project programmatically? If so, then maybe we should create one and test that the result contains that project?